dat/events/neutral/npc.lua

```diff
@@ -248,8 +248,8 @@ function create()
 
    -- Chance of a jump point message showing up. As this gradually goes
    -- down, it is replaced by lore messages. See spawnNPC function below.
-   jm_chance_min = 0
-   jm_chance_max = 0.25
+   jm_chance_min = 0.25
+   jm_chance_max = 0.35
```

Comment:
This seems unrelated to the topic of the pull request, no? I disagree with the change; I intentionally made it so that jump messages disappear both to minimize the damaging "clickfest" effect it can create (confine it to the start of the game) and so that you don't get bothered with unnecessary garbage later on in the game. You've in fact changed it here so that jump messages are more common than they ever used to be even at its lowest IIRC.